### ISSUE TITLE
Hard to read

### ISSUE THREAD

**Issue by kc2rxo** [ietf-wg-drip/draft-ietf-drip-auth#111]
I found the following sentence a little hard to parse - it likely could be made
easier to read?: “Note however that if Length octets was exhausted
   exactly at the end of an Authentication Page then the Additional Data
   Length field will occupy the first octet of the following page the
   remainder of which under DRIP will be null padded.”

**Comment by rgmhtt** [comment]
First I think we need to change 3.2.2 and give the exact Authentication Data / Signature length per the length field in the Authentication Header.  This is known and so no guessing is needed in 5.2 to find the ADL field if present and then FEC if present.

Then:

5.2.  Decoding

Frame decoding is independent of the transmit media. However the decoding process can determine from the first Authentication page that there may be a Bluetooth 4.X FEC page at the end.  The decoding process MUST test for the presence of FEC and apply it as follows.

If Modulo(Authentication Data length + 6, n? (note I don't know this # for here) is [1- n-1], the ADL is in the last Authentication page containing the Authentication Data / Signature, otherwise the ADL is in the next page, if present.  The ADL is checked and if...

And I think you can compete the rest better than me.
